### DISCUSSION TITLE
AdjustmentFilter not working for me

### DISCUSSION THREAD

**Discussion by steven-twerdochlib** [pixijs/pixijs#10768] [Q&A]
Hi,
I'm currently upgrading from pixi v7 to v8 but the follwoing code isn't working:
```
const filter = new AdjustmentFilter(filterOptions);
let tmpFilter = [...randomObject.image.filters];
tmpFilter.push(filter);
randomObject.image.filters = tmpFilter;
```
Note that the below code works for inverting the image:
```
const negativeFilter = new PIXI.ColorMatrixFilter();
negativeFilter.negative(true);
let tmpFilter = [...randomObject.image.filters];
tmpFilter.push(negativeFilter);
randomObject.image.filters = tmpFilter;
```
I don't see what the difference is between the two, has PIXI.AdjustmentFilter changed in some way for v8?

**Comment by godofecht** [comment]
I've just run into the same issue.
Can replicate, and this is quite odd.
